shared/elements/gaia_lowstorage_dialog/examples/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
```

Comment:
__question:__ do we have convention when we use underscore and dashes in the component name, I see we have both in shared folder? I remember Wilson told me that underscore is used for old/kind-of-obsoleted components.